packages/key-readmodel-writer/src/index.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable functional/immutable-data */
```

Comment:
```suggestion
```
Just now noticing this, it shouldn't be needed but it's present in the others too

Reply:
Removed